icu4c/source/i18n/unitsdata.cpp

```diff
@@ -107,8 +107,8 @@ class ConversionRateDataSink : public ResourceSink {
     MaybeStackVector<ConversionRateInfo> *outVector;
 };
 
-UnitPreferenceMetadata::UnitPreferenceMetadata(const char *category, const char *usage,
-                                               const char *region, int32_t prefsOffset,
+UnitPreferenceMetadata::UnitPreferenceMetadata(const StringPiece category, const StringPiece usage,
```

Comment:
Optional: there's no need to make a `const StringPiece`, since StringPiece is passed by value.

Reply:
Done.